animate/metric.py

```diff
@@ -128,19 +133,58 @@ def _process_parameters(self, metric_parameters):
         return mp, vp
 
     def set_parameters(self, metric_parameters={}):
```

Comment:
I think it would be useful to raise a warning if the user tries to set a parameter which doesn't exist or isn't supported. In case of typos - which I think are quite likely since parameter names are quite long, like `dm_plex_metric_target_complexity`...

Reply:
It happened to me multiple times actually and it's always annoying to debug

Reply:
Good idea. Fixed in 3ae4d9e900362bd0adb22db1848c1623c6b769d6.